Exercise_C/index.css

```diff
@@ -2,3 +2,30 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.boxContainer{
+    display: flex;
+    justify-content: space-between;
+}
+
+.box{
+    height: 100px;
+    width: 100px;
+}
+
+#blue{
+    background-color: blue;
+}
+
+#green{
+    background-color: green;
+}
+
+.boxContainerRed{
```

Comment:
Very smart to use two containers to force the boxes onto the correct line, great work.